src/index.js

```diff
@@ -1,4 +1,151 @@
+// /* eslint-disable no-unused-expressions */
+// expect(someTrueValue).to.be.true;
+
```

Comment:
- [ ] Please make sure to delete the comments lines of code in your file to keep a clean codebase 🙃🙃